### ISSUE TITLE
repeater using class names as invalid selector with `querySelector`

### ISSUE THREAD

**Issue by smakinson** [formkit/formkit#1417]
### Reproduction

https://formkit.link/4acfdfbc80d4d929099d96d1dd56cf09

### Describe the bug

NOTE: The playground link does NOT reproduce the issue, but tried to give an idea of usage.

This is the issue @sashamilenkovic requested in the discussion here:
https://discord.com/channels/793529058377072650/1250864751580545034

The error is: 
```
SyntaxError: Failed to execute 'querySelector' on 'Document': '#myRepeater .formkit-inner bg-neutral-50 dark:bg-neutral-900 text-gray-950 dark:text-gray-100 rounded formkit-disabled:bg-gray-100 dark:formkit-disabled:bg-gray-950 formkit-disabled:cursor-not-allowed formkit-disabled:pointer-events-none dark:formkit-disabled:text-gray-100' is not a valid selector."
```

@fenilli suggested it is related to this line:
```
  const innerSection = node.props.__root?.querySelector(
    `#${node.props.id} .${node.context.classes.inner}`
  )
```

### Environment

• OS: macos
• Browser chrome
• Version 125.0.6422.142

**Comment by yhtomitim** [comment]
Is there a workaround for this at this time?

**Comment by sashamilenkovic** [comment]
@smakinson @yhtomitim Apologies on the delay for this! This should now be resolved at FormKit Pro v0.126.13. Let me know if you're still having any issues.

**Comment by smakinson** [comment]
@sashamilenkovic Thank you for this fix. It looks like it took care of the issue.
